samples/delete_analysis.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+from google.cloud.contact_center_insights_v1.services.contact_center_insights import client
+
+
+# [START contactcenterinsights_delete_analysis]
+def delete_analysis(name: str) -> None:
+    insights_client = client.ContactCenterInsightsClient()
+    insights_client.delete_analysis(name=name)
+    print("Deleted an analysis named {}".format(name))
+
+# [START contactcenterinsights_delete_analysis]
```

Comment:
This should be `[END contactcenterinsights_delete_analysis]`

Reply:
Oops copy/paste mistake. Fixed!